azurerm/internal/services/containers/container_registry_scope_map_resource.go

```diff
@@ -37,10 +37,9 @@ func resourceContainerRegistryScopeMap() *schema.Resource {
 
 		Schema: map[string]*schema.Schema{
 			"name": {
-				Type:         schema.TypeString,
-				Required:     true,
-				ForceNew:     true,
-				ValidateFunc: validate.ContainerRegistryName,
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
```

Comment:
rather than removing this, can we add a validator which includes the correct values here?

Reply:
Done, added in a new validator & tests for scope maps